File: py/9.py
for a in xrange(5, 1000, 5):
    done = False

    for b in xrange(5, 1000 - a, 5):
        c = 1000 - a -b
        if a * a + b * b == c * c:
            print a * b * c
            done = True

    if done:
        break

File: rb/9.rb
(5..1000).step(5).each do |a|
  done = false

  (5..1000 - a).step(5).each do |b|
    c = 1000 - a - b
    done = p a * b * c if a * a + b * b == c * c
  end

  break if done
end
